src/components/QuizQuestion.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="quiz__container">
+    <div class="quiz__image-box--form" />
+    <div class="quiz__question-box">
+      <div class="quiz__question-info">
+        <h1 class="quiz__step">Question {{ step }}</h1>
+        <slot name="wizard" />
+        <span class="quiz__question">{{ question.value }}</span>
+      </div>
+      <component
+        :is="answerType"
```

Comment:
What do you think about calling `answerType` a bit more descriptively, so it's clear it indicates a component name, like `answerComponent`?